netbox/ipam/api/views.py

```diff
@@ -143,6 +143,18 @@ class VLANViewSet(NetBoxModelViewSet):
     filterset_class = filtersets.VLANFilterSet
 
 
+class VLANTranslationPolicyViewSet(NetBoxModelViewSet):
+    queryset = VLANTranslationPolicy.objects.prefetch_related('rules')
```

Comment:
Prefetching should be handled dynamically for the requested fields.
```suggestion
    queryset = VLANTranslationPolicy.objects.all()
```